dali/operators/reader/loader/loader.h

```diff
@@ -317,8 +338,9 @@ class Loader {
   std::once_flag fetch_cache_;
   std::shared_ptr<ImageCache> cache_;
 
-  // Counts how many samples reader have read already from this and next epoch
+  // Counts how many samples reader have read already from this epoch
   Index read_sample_counter_;
+  Index returned_sample_counter_;
```

Comment:
Can you describe this member variable as well?

Reply:
Done